### ISSUE TITLE
support Django 4.2 and drf 3.14???

### ISSUE THREAD

**Issue by whitewhite944** [rsinger86/drf-flex-fields#130]

**Comment by ihoru** [comment]
The package actually works just fine with these two.

**Comment by mikedvinci90** [comment]
@ihoru, could you please confirm whether you are certain that this package will operate smoothly with async views and APIs? As far as I know, even DRF has separate async serializers, as mentioned here: (https://github.com/em1208/adrf#async-serializers). I'm not seeing async supported code in this package.

**Comment by ihoru** [comment]
Nope, I can't confirm that. I only tested it in the synchronous environment. 

> @ihoru, could you please confirm whether you are certain that this package will operate smoothly with async views and APIs? As far as I know, even DRF has separate async serializers, as mentioned here: (https://github.com/em1208/adrf#async-serializers). I'm not seeing async supported code in this package.

**Comment by mweinelt** [comment]
The tests also don't fun fine, given that they want to import `url` from `django.conf.urls`, which was deprecated already deprecated in 3.2.

https://docs.djangoproject.com/en/3.2/_modules/django/conf/urls/
